src/Main.hs

```diff
@@ -83,10 +84,23 @@ initDbAndFetcher env Config {..} =
               Right () -> pure ()
               Left e -> logErrorNS "initDbAndFetcher" $ Text.pack $ show e
 
+argsParse :: [String] -> String -> String -> String
+argsParse args key vdefault = case args of
+  [] -> vdefault
+  arg:next -> fromMaybe (argsParse next key vdefault) (stripPrefix (key++"=") arg)
+  where
+  --https://hackage.haskell.org/package/base-4.16.1.0/docs/src/Data-OldList.html#stripPrefix
+  stripPrefix :: Eq a => [a] -> [a] -> Maybe [a]
+  stripPrefix [] ys = Just ys
+  stripPrefix (x:xs) (y:ys)
+    | x == y = stripPrefix xs ys
+  stripPrefix _ _ = Nothing
```

Comment:
I think here it will be better to do it in one of the following ways:

- simplify this code to parse arguments by pattern matching (`getArgs >>= \case [] -> ...; [config] -> ...`), which is pretty good while we don't need a complex CLI interface
- or use some library to parse CLI, which generates `--help` message for us and will take into account all corner cases (have not a bad experience with [cmdargs](https://hackage.haskell.org/package/cmdargs-0.10.21/docs/System-Console-CmdArgs-Implicit.html))

Reply:
Agreed, we should definitely use a proper CLI interface. I'm a fan of opt-parse applicative, since plutus projects already have it.